tenets/codelingo/go/ticker-in-for-switch/codelingo.yaml

```diff
@@ -1,12 +1,39 @@
+funcs:
+  - name: extractRaw
+    type: resolver
+    body: |
+      function(rawCode) {
+        var splitCode = rawCode.split("\n")
+        var re = /case <-time\.After\(\w*/
+        var blank = splitCode[splitCode.length -1]
+        blank = blank.substring(0, blank.indexOf("}"))
+
+        for (var i = 0; i < splitCode.length; i++) {
+        	line = splitCode[i]
+        	if (line.match(re)) {
```

Comment:
Not sure why this indentation is messed up. Looks fine on my PC

Reply:
tabs v spaces?